File: backend/src/main/java/com/swarm/feedback/service/FeedbackService.java
package com.swarm.feedback.service;

import com.swarm.feedback.model.Feedback;
import java.util.List;

public interface FeedbackService {
    Feedback createFeedback(Feedback feedback);

    List<Feedback> getFeedbackBySubmissionId(String submissionId);

    List<Feedback> getFeedbackByReviewerId(String reviewerId);

    List<Feedback> getAllFeedback();
}

File: frontend/src/services/feedback.service.js
import api from './api';

const getAllFeedback = () => {
    return api.get('/feedback');
};

const createFeedback = (feedback) => {
    return api.post('/feedback', feedback);
};

const FeedbackService = {
    getAllFeedback,
    createFeedback,
};

export default FeedbackService;
